flash/vision/classification/model.py

```diff
@@ -58,14 +58,12 @@ class ImageClassifier(ClassificationTask):
         learning_rate: Learning rate to use for training, defaults to ``1e-3``.
     """
 
-    @property
-    def preprocess(self):
-        return ImageClassificationPreprocess(predict_transform=ImageClassificationData.default_val_transforms())
-
     def __init__(
         self,
         num_classes: int,
         backbone: Union[str, Tuple[nn.Module, int]] = "resnet18",
+        backbone_kwargs: Dict = {},
+        head: Optional[Union[Callable, nn.Module]] = None,
         pretrained: bool = True,
```

Comment:
Maybe out of scope of this PR but why do we have by default to True ? since this downloads from server, it might cause issues like we saw in Kaggle or MNIST. Even for testing might be more safe.

Reply:
@tchaton I think backbone_kwargs should be None here. We should not have mutable default args

Reply:
Done !